pkg/workload/cli/run.go

```diff
@@ -354,7 +354,7 @@ func runRun(gen workload.Generator, urls []string, dbName string) error {
 	var ops workload.QueryLoad
 	prepareStart := timeutil.Now()
 	log.Infof(ctx, "creating load generator...")
-	const prepareTimeout = 10 * time.Minute
```

Comment:
optional nit: perhaps a small TODO here would be useful so that we write down the fact that this is a temporary measure and that we eventually want to reduce this timeout again.

Reply:
I'm not actually sure that we do want to drop it down again. 10 minutes was pretty arbitrary and if it's causing issues then we should keep it bumped up. I'm skeptical that we should have a timeout here to begin with, but I guess it leads to better failure cases when things really do get stuck.